invideoquiz/invideoquiz.py

```diff
@@ -55,7 +56,9 @@ class InVideoQuizXBlock(StudioEditableXBlockMixin, XBlock):
         help=_(
             'A simple string field to define problem IDs '
             'and their time maps (in seconds) as JSON. '
-            'Example: {"60": "50srvqlii4ru9gonprp35gkcfyd5weju"}'
+            'Example: {"60": "50srvqlii4ru9gonprp35gkcfyd5weju"} '
+            'Problem IDs can be obatined from staff debug info of '
+            'the problems in LMS.'
```

Comment:
Trivia: This was much easier to do on the Stanford fork because we modified Studio to always display the location ID in the footer of components (Studio only). This made it easy to see without any extra debugging. Unfortunately, edx rejected that change :cry: